provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -30,35 +30,162 @@ includes:
     internal: true
 
 tasks:
+  fetch-rosa-region:
+    internal: true
+    requires:
+      vars:
+        - ROSA_CLUSTER_NAME
+    cmd: rosa describe cluster -c {{ .ROSA_CLUSTER_NAME }} -o json | jq -r .region.id > .task/rosa-region-{{.ROSA_CLUSTER_NAME}}
+    generates:
+      - .task/rosa-region-{{.ROSA_CLUSTER_NAME}}
 
   create-aurora:
+    cmds:
+      - task: create-{{ ternary "global" "single" (eq .CROSS_DC_MODE "ASYNC") }}-aurora
```

Comment:
Personally I would use a specific env variable here, maybe something like `AURORA_MODE` or `AURORA_DEPLOYMENT`, given that ASYNC multi-az deployments might be resurrected ... but I'm happy to stick with this if you think it'll be flexible enough for our needs.